docs/guides/es2k/enabling-comms-channel.md

```diff
@@ -0,0 +1,131 @@
+# Enabling the COMMS Channel
```

Comment:
We should keep this documentation consistent with other places. Is this the terminology used in other customer documentation? Calling it `COMMS channel` (in uppercase and shortened form)?

Reply:
Communication channels(comm_chnnls) is used, will modify to use this instead